classes/maps.php

```diff
@@ -82,11 +80,8 @@ function set_map_atts( $atts = array() ) {
 			// Do atts for iframes
 			if ( isset( $_GET['sm_map_iframe'] ) ) {
 				// Set atts from GET vars
-				if ( ! empty( $_GET['map_width'] ) ) {
-					$atts['map_width'] = $_GET['map_width'];
-				}
-				if ( ! empty( $_GET['map_height'] ) ) {
-					$atts['map_height'] = $_GET['map_height'];
+				if ( ! empty( $_GET['api_key'] ) ) {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->